llama_stack/core/server/server.py

```diff
@@ -236,6 +237,10 @@ async def route_handler(request: Request, **kwargs):
                     result = await maybe_await(value)
                     if isinstance(result, PaginatedResponse) and result.url is None:
                         result.url = route
+
+                    if method.upper() == "DELETE" and result is None:
+                        return Response(status_code=httpx.codes.NO_CONTENT)
```

Comment:
thanks. suggestion: as another tiny PR, there are two other codes in this file (426 and 429 I think?) which could use a httpx.codes conversion.